src/suite-design-guide/general-principles.rst

```diff
@@ -104,9 +104,24 @@ files they need are more robust.
 Installing Files At Start-up
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use ``rose suite-run`` *file creation mode* or ``R1``
-install tasks to copy files to the self-contained suite run directory at
-start-up. Install tasks are preferred for time-consuming installations because
+Files can be installed on the remote platform. As standard, Cylc installs the
+following directories: ``app``, ``bin``, ``etc``, ``lib``.
+Cylc supports adding custom directories and files to the file installation.
+E.g. dir1, dir2, file1, file2, you can add the following configuration to your
+:cylc:conf:`flow.cylc`. To denote a directory, please add a trailing slash.  
```

Comment:
Could you squeeze in a reference to :cylc:conf:`flow.cylc[scheduler]install` so that people have a nice easy link into the relevant bit of the reference docs.